lib/interop.js

```diff
@@ -196,15 +196,15 @@ function createSourceGroupMap(sourceGroups) {
  * @returns {Boolean} - Returns true if an ICE restart is requested otherwise false.
  */
 function checkForIceRestart(newDesc, oldDesc) {
-    if (!newDesc || !oldDesc ||
-        newDesc.media.length === 0 || oldDesc.media.length === 0) {
+    if (!newDesc || !oldDesc
+        || newDesc.media.length === 0 || oldDesc.media.length === 0) {
```

Comment:
We enforce a max line length of 120 instead of 80. Do you think all this will fit in a single line ?